packages/lesswrong/components/linkPreview/PostLinkPreview.tsx

```diff
@@ -642,7 +697,7 @@ const ManifoldPreview = ({classes, href, id, children}: {
   const isEmbed = /^https?:\/\/manifold\.markets\/embed\/.+$/.test(href);
 
   // if it fits  https://manifold.markets/[username]/[market-slug] instead, get the (username and market slug)
-  const [, userAndSlug] = href.match(/^https?:\/\/manifold\.markets\/(\w+\/[\w-]+)/) || [];
+  const [userAndSlug] = href.match(/^https?:\/\/manifold\.markets\/(\w+\/[\w-]+)/) || [];
```

Comment:
This is probably a bug - `userAndSlug` will now be equal to "https://manifold.markets/[username]/[market-slug]" instead of just "[username]/[market-slug]"

Reply:
I was sort of confused what was going on here – could @s-cheng explain it?

Reply:
I'm not familiar with this code, perhaps it was just confusing because Sinclair Chen's name is similar to mine :) Apologies I don't really have time to help with this, so if you want another opinion I suggest asking @b0b3rt

Reply:
`match` returns an array of strings - index 0 is the complete match and the following items are the capture groups. In this case we want the user and slug which is the first capture group so it will be at index 1. Having that comma in the destructuring means that `userAndSlug` will bind to index 1, without the comma `userAndSlug` will bind to index 0 (the full match).

Reply:
Oh sorry, yep that was just a misread of Sinclair Chen, sorry about that. Thanks Ollie – I've previously seen destruction with, say, an underscore as the first variable, but hadn't seen the pattern with the raw comma.